dt-core/admin/components/setup-wizard-use-cases.js

```diff
@@ -122,46 +122,55 @@ export class SetupWizardUseCases extends OpenLitElement {
             : ''}
           ${this.stage === 'work'
             ? html`
-                <h2>Choose a use case</h2>
+                <h2>Part 1: Use Cases</h2>
```

Comment:
all these h2's could they go outside the conditional rendering? like the other steps do?